installer/u-boot-arch/install-arch

```diff
@@ -67,6 +67,7 @@ update_uboot_env()
 
     (cat <<EOF
 onie_boot_reason
+onie_nos_mode
```

Comment:
u-boot arch does not have embed mode.  Is it necessary to cleared `onie_nos_mode` after upgrading ONIE itself? (`nos_bootcmd` is not cleared here)

Reply:
good point.